intermediate-python-debugging/exercise-using-debugger.md

```diff
@@ -0,0 +1,76 @@
+# Problem Set: Using the Debugger
+
+In this exercise we will use the debugger to identify two errors.  To run the activity clone the code at [AdaGold/python-debugging](https://github.com/AdaGold/python-debugging).  Then set up the project by creating a virtual environment and installing pytest.
+
+The code contains two bugs, one runtime error crashes the program on execution and the other is a logic error which calculates the incorrect sum for an Order instance.  
+
+## Activity
+
+### 1st Bug
```

Comment:
```suggestion
### First Bug: Debugging with Breakpoints
```

Grammar. Suggesting adding ": Debugging with Breakpoints" to give it more context that helps frame the point of the first bug to students